include/fx/gltf.h

```diff
@@ -965,11 +965,12 @@ namespace gltf
                 std::string jsonText = json.dump();
 
                 Buffer const & binBuffer = document.buffers.front();
-                const uint32_t paddedLength = ((binBuffer.byteLength + 3) & (~3u));
-                const uint32_t padding = paddedLength - binBuffer.byteLength;
-                binHeader.chunkLength = paddedLength;
+                const uint32_t binPaddedLength = ((binBuffer.byteLength + 3) & (~3u));
+                const uint32_t binPadding = binPaddedLength - binBuffer.byteLength;
+                binHeader.chunkLength = binPaddedLength;
 
-                header.jsonHeader.chunkLength = jsonText.length() & 0xffffffffu;
+                header.jsonHeader.chunkLength = ((jsonText.length() + 3) & (~3u));
+                const uint32_t headerPadding = (uint32_t)(header.jsonHeader.chunkLength-jsonText.length());

```

Comment:
Can you use static_cast instead of c-style casts and add a space between the '-'

Reply:
Wow, I'm so old-school (wacom C++ days) I had no idea about the modern C++ static_cast until I read up on it.  Thanks for the tip.